src/System.Net.Mail/tests/Functional/AlternateViewCollectionTest.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+//
+// AlternateViewCollectionTest.cs - NUnit Test Cases for System.Net.MailAddress.AlternateViewCollection
+//
+// Authors:
+//   John Luke (john.luke@gmail.com)
+//
+// (C) 2005 John Luke
```

Comment:
@AlexGhiondea, I'm wondering if it's sufficient just to keep the actual copyright from the file, rather than all comments that were at the top of the file?  i.e. seems like we could just keep:
```C#
// (C) 2005 John Luke
```
rather than
```
// AlternateViewCollectionTest.cs - NUnit Test Cases for System.Net.MailAddress.AlternateViewCollection
//
// Authors:
//   John Luke (john.luke@gmail.com)
//
// (C) 2005 John Luke
```
These aren't NUnit tests, and we don't need email addresses like this.  Seems like we should only be carrying forward the actual copyright/permissions sections, and not everything else?